src/index.html

```diff
@@ -1,13 +1,57 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+          crossorigin = "use-credentials">
+    <link href="https://fonts.googleapis.com/css2&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>MOYO product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+
+    <div class="goodsCard" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        class="goodsCard__image"
+
+        alt="imac"
+      >
+      <p class="goodsCard__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+      <p class="goodsCard__article">
```

Comment:
```suggestion
      </p>
      
      <p class="goodsCard__article">
```

Add one empty space between each siblings blocks inside same parent block